doc/CCPPtechnical/source/CodeManagement.rst

```diff
@@ -162,7 +161,7 @@ Start with checking out the main repository from the NCAR GitHub
 
 .. code-block:: console
 
-   git clone -b gmtb/ccpp https://github.com/NCAR/NEMSfv3gfs
+   git clone -b https://github.com/NCAR/NEMSfv3gfs
    cd NEMSfv3gfs
    git submodule init
    git submodule update
```

Comment:
I can answer this one. NEMS contains a nested submodule, which at this point still lives on Vlab. The code of this submodule is not required for the "normal" user (only when running the NEMSCompset regression test version, which is different from the regression tests run through rt.sh). Because it is in Vlab, a `--recursive` leads to a checkout error. Doing it the way Ligia described it avoids this problem.